input/blog/2021-08-29-cake-v1.2.0-released.md

```diff
@@ -0,0 +1,93 @@
+- [__#2578__](https://github.com/cake-build/cake/issues/2578) Feature request: nuget version ranges support.
+- [__#2362__](https://github.com/cake-build/cake/issues/2362) Add Support for New snupkg Symbol Packages.
+
+__Improvement__
+
+- [__#3429__](https://github.com/cake-build/cake/issues/3429) Microsoft.Extensions.DependencyInjection to 5.0.2.
+- [__#3427__](https://github.com/cake-build/cake/issues/3427) Update Microsoft.CodeAnalysis.CSharp.Scripting to 3.11.0.
+- [__#3425__](https://github.com/cake-build/cake/issues/3425) Update NuGet Client libraries to 5.11.0.
+- [__#3423__](https://github.com/cake-build/cake/issues/3423) Update Spectre.Console to 0.41.0.
+- [__#3337__](https://github.com/cake-build/cake/issues/3337) Suppress compilation warnings CS1701, CS1702, and CS1705.
+- [__#3316__](https://github.com/cake-build/cake/issues/3316) Bump NuGet client libraries to 5.9.1.
+- [__#3314__](https://github.com/cake-build/cake/issues/3314) Bump .NET SDK to 5.0.202.
+- [__#3294__](https://github.com/cake-build/cake/issues/3294) Clean up task builder extensions.
+- [__#3281__](https://github.com/cake-build/cake/issues/3281) GitVersion Tool: Remap existing verbosity values to valid GitVersion values.
+- [__#3255__](https://github.com/cake-build/cake/issues/3255) Update NuGet client libraries to 5.9.0.
+- [__#3253__](https://github.com/cake-build/cake/issues/3253) Update Microsoft.CodeAnalysis.CSharp.Scripting to 3.9.0 stable.
+- [__#3246__](https://github.com/cake-build/cake/issues/3246) Update Spectre.Console to 0.38.0.
+- [__#3223__](https://github.com/cake-build/cake/issues/3223) Feature request: Environment variable substitution in cake.config.
+- [__#2654__](https://github.com/cake-build/cake/issues/2654) NUnit3Settings should support TestParam.
+- [__#2168__](https://github.com/cake-build/cake/issues/2168) TypeConverter to enable Argument<DirectoryPath>(...).
+- [__#2030__](https://github.com/cake-build/cake/issues/2030) NuGetHasSource is case sensitive.
+
+__Documentation__
+
+- [__#3365__](https://github.com/cake-build/cake/issues/3365) Typo in documentation of NuGetAdd alias.
+- [__#3355__](https://github.com/cake-build/cake/issues/3355) VSTest alias documentation contains holdover from <v0.17.0.
+- [__#3283__](https://github.com/cake-build/cake/issues/3283) Update GitVersion alias reference page with dotnet tool usage example.
+- [__#3259__](https://github.com/cake-build/cake/issues/3259) TypeLoadException: Missing implementation of RegisterLazy.
+
+__Build__
+
+- [__#3431__](https://github.com/cake-build/cake/issues/3431) Update Microsoft.NET.Test.Sdk to 16.11.0.
+- [__#3421__](https://github.com/cake-build/cake/issues/3421) Update .NET SDK to 5.0.400.
+- [__#3372__](https://github.com/cake-build/cake/issues/3372) Update Microsoft.NET.Test.Sdk to 16.9.4.
+- [__#3370__](https://github.com/cake-build/cake/issues/3370) Update Spectre.Console to 0.39.0.
```

Comment:
Shouldn't this be listed under improvements, like other Spectre.Console updates?

Reply:
Sure, it's based on issue labels.
I'll take a look.